src/components/CategoryFilters.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div
+    class="bg-white text-xs gap-6 p-5 rounded-md shadow-sm mb-12 overflow-x-auto"
+  >
+    <ul class="flex mr-6">
```

Comment:
Aqui a margem não está fazendo nada:
```suggestion
    <ul class="flex">
```

Reply:
Ok